src/config/entities/__tests__/configuration.ts

```diff
@@ -84,20 +84,32 @@ export default (): ReturnType<typeof configuration> => ({
       apiKey: faker.string.hexadecimal({ length: 32 }),
     },
   },
-  typeorm: { autoLoadEntities: true, manualInitialization: true },
   db: {
-    postgres: {
-      schema: process.env.POSTGRES_SCHEMA || 'main',
-      host: process.env.POSTGRES_TEST_HOST || 'localhost',
-      port: process.env.POSTGRES_TEST_PORT || '5433',
-      database: process.env.POSTGRES_TEST_DB || 'test-db',
-      username: process.env.POSTGRES_TEST_USER || 'postgres',
-      password: process.env.POSTGRES_TEST_PASSWORD || 'postgres',
-      ssl: {
-        enabled: true,
-        requestCert: true,
-        rejectUnauthorized: true,
-        caPath: process.env.POSTGRES_SSL_CA_PATH,
+    migrator: {
+      migrationsExecute: true,
```

Comment:
Nit: it's semantically clearer with the verb first:

```suggestion
      executeMigrations: true,
```

Reply:
Question: Do we have a naming convention? The former emphasizes the action of migration.
I'll fix it soon.

Reply:
We don't - it was more grammatical. If you prefer it as is, it doesn't need changing.